pkg/api/dashboard.go

```diff
@@ -326,6 +326,12 @@ func (hs *HTTPServer) deleteDashboard(c *contextmodel.ReqContext) response.Respo
 		hs.log.Error("Failed to disconnect library elements", "dashboard", dash.ID, "user", c.SignedInUser.UserID, "error", err)
 	}
 
+	// deletes all related public dashboard entities
+	err = hs.PublicDashboardsApi.PublicDashboardService.DeleteByDashboard(c.Req.Context(), dash)
+	if err != nil {
+		hs.log.Error("Failed to delete public dashboard")
+	}
```

Comment:
shouldn't we return the error?

Reply:
no, I don't want to block people from deleting dashboards